.gitignore

```diff
@@ -1,24 +1,14 @@
-# Don't put *.swp, *.bak, etc here; those belong in a global ~/.gitignore.
+# Don't put *.swp, *.bak, etc here; those belong in a global ~/.gitignore_global.
```

Comment:
The line below provides a link.
I just want to be consistent with content that the link provides https://help.github.com/articles/ignoring-files.